src/sub/app.py

```diff
@@ -186,3 +186,4 @@ def after_request(response):
     app.debug = True
     app.use_reloader = True
     app.run(host="0.0.0.0", port=CFG.LOCAL_FLASK_PORT)
+    logger.info("sub running", app=app)
```

Comment:
don't want the same thing in hub/app.py?

Reply:
Yes, changing.